packages/components/src/logos/index.tsx

```diff
@@ -23,27 +23,37 @@ export { ReactComponent as HiveCombinationMark } from './hive-combination-mark.s
 export { ReactComponent as GraphQLFoundationLogo } from './graphql-foundation.svg';
 
 export interface LettermarkLogoProps extends HTMLAttributes<HTMLElement> {}
-const createLettermarkLogo = (text: string) => {
+
+const HOCLettermarkLogo = (text: string) =>
```

Comment:
HOC is better name

Reply:
I don't want to block the PR for this, but it's technically incorrect.

https://legacy.reactjs.org/docs/higher-order-components.html
https://en.wikipedia.org/wiki/Higher-order_function

A higher order X is something that takes X and returns X.

This function takes a string and returns a component, so it might be a "component factory" if you really want to use a noun for it.

https://en.wikipedia.org/wiki/Factory_(object-oriented_programming)

Reply:
thank you, yeah HOC should take the component as an argument 👍